src/coreclr/vm/jitinterface.cpp

```diff
@@ -10836,17 +10850,19 @@ void CEECodeGenInfo::getHelperFtn(CorInfoHelpFunc    ftnNum,               /* IN
         {
             accessType = IAT_VALUE;
             targetAddr = finalTierAddr;
-            if (pMethod != nullptr && HasILBasedDynamicJitHelper(dynamicFtnNum))
+            if (pMethod != NULL && HasILBasedDynamicJitHelper(dynamicFtnNum))
             {
-                (void)LoadDynamicJitHelper(dynamicFtnNum, &helperMD);
+                (void)LoadDynamicJitHelper(dynamicFtnNum);
```

Comment:
```suggestion
```
GetMethodDescForILBasedDynamicJitHelper will load the MethodDesc lazily as necessary